nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/util/SecureNiFiConfigUtil.java

```diff
@@ -117,6 +123,17 @@ public static void configureSecureNiFiProperties(final String nifiPropertiesFile
                 String[] subjectAlternativeNames = getSubjectAlternativeNames(nifiProperties, cmdLogger);
                 tlsConfiguration = KeyStoreUtils.createTlsConfigAndNewKeystoreTruststore(StandardTlsConfiguration
                         .fromNiFiProperties(nifiProperties), CERT_DURATION_DAYS, subjectAlternativeNames);
+                final KeyStore keyStore = KeyStoreUtils.loadKeyStore(tlsConfiguration.getKeystorePath(),
+                        tlsConfiguration.getKeystorePassword().toCharArray(), tlsConfiguration.getKeystoreType().getType());
+                final Enumeration<String> aliases = keyStore.aliases();
+                while (aliases.hasMoreElements()) {
+                    final String alias = aliases.nextElement();
+                    final Certificate certificate = keyStore.getCertificate(alias);
+                    if (certificate != null) {
+                        final byte[] sha256 = DigestUtils.getSha256Digest().digest(certificate.getEncoded());
+                        cmdLogger.info("SHA256(self-signed-certificate) = {}", Hex.encodeHexString(sha256).toUpperCase(Locale.ROOT));
```

Comment:
Making the log message follow a similar pattern as the expiration message would be helpful:
```suggestion
                        cmdLogger.info("Generated Self-Signed Certificate SHA-256: {}", Hex.encodeHexString(sha256).toUpperCase(Locale.ROOT));
```